crates/router/src/db/ephemeral_key.rs

```diff
@@ -64,7 +64,7 @@ mod storage {
                 .serialize_and_set_multiple_hash_field_if_not_exist(
                     &[(&secret_key, &created_ek), (&id_key, &created_ek)],
                     "ephkey",
-                    None,
+                    Some(validity.saturating_mul(60).saturating_mul(60)),
```

Comment:
This validity is for hours, so minimum validity is 1 hour and can be configured in steps of hours